### PR TITLE
Create deriveapprodi.csl

### PR DESCRIPTION
**Pull request by tubia**
Created the csl for the Italian publisher DeriveApprodi

### CODE REVIEW

File: deriveapprodi.csl [hunk 1]
```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="it-IT">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>DeriveApprodi (Italiano)</title>
+    <id>http://www.zotero.org/styles/deriveapprodi</id>
+    <link href="http://www.zotero.org/styles/deriveapprodi" rel="self"/>
+    <link href="http://www.zotero.org/styles/geografie-sbornik-cgs" rel="template"/>
+    <link href="https://forums.zotero.org/discussion/97879/style-request-deriveapprodi" rel="documentation"/>
```
Comment (POBrien333) [inline deriveapprodi.csl:9]: There is no publicly available URL for the citation guidelines?

Comment (tubia) [reply deriveapprodi.csl:9]: Hi, this style is based on the citation guidelines I got from the publisher, that are sent to the prospect authors. Do they have to be publicly available?

**Comment by tubia** [comment]
Any news?

**Comment by adam3smith** [comment]
Sorry for the delay here -- we're generally a bit reluctant to accept styles where we can't check the author guidelines (both on initial submission but especially in case there are error reports later on). Would you be willing to upload the document you received here?

**Comment by adam3smith** [comment]
@tubia please see my question above

**Comment by adam3smith** [comment]
Closing for inactivity -- happy to open if you come back to this, just post a short message here.

**Pull request closed without merging**